### ISSUE TITLE
Hours worked added to Department (Staffer Summary)

### ISSUE THREAD

**Issue by bds002** [magfest/ubersystem#1802]
If you look at: https://prime.uber.magfest.org/uber/jobs/staffers?location=216073079

We have each person in the department list and it list the shifts hours they are signed up for both in the department and in other departments.  I would like to see added a another column to add the Worked Hours for each staffer.  We are already making that calculation and it can be found here:
https://prime.uber.magfest.org/uber/registration/staffers

So this should just be a matter of adding the correct script to a new column.

**Comment by EliAndrewC** [comment]
The additional column should only appear when we're at the con or post-con, i.e. when ``c.AT_OR_POST_CON`` is truthy.

**Comment by ChronoAndross** [comment]
Is this still a relevant issue? I would like to take a crack at it. Would you know how to reproduce this bug in a vagrant build? I don't have admin access, so I am using a vagrant deploy at the moment.

Thanks!
Andrew

**Comment by bds002** [comment]
Yes it is.  Relevant in that it would be used in the next deploy version (Classic etc)
